### PR TITLE
Include markers for test cases based on priority

### PR DESCRIPTION
**Pull request by yarunachalam**
- Include markers for test cases based on priority.
- Include comment with test case number and description

### CODE REVIEW

File: harvester_e2e_tests/apis/test_images.py [hunk 1]
```diff
@@ -27,12 +27,17 @@
...
+@pytest.mark.p1
 def test_list_images(admin_session, harvester_api_endpoints):
     resp = admin_session.get(harvester_api_endpoints.list_images)
     assert resp.status_code == 200, (
         'Failed to list images: %s' % (resp.content))
 
 
+@pytest.mark.images_p2
+@pytest.mark.p2
+# This negative test covers images-3-Create Image no data
```
Comment (guangyee) [inline harvester_e2e_tests/apis/test_images.py:40]: Lets use python docstring format for these. https://www.python.org/dev/peps/pep-0257/#toc-entry-6

For example,

```
def test_create_image_no_data(admin_session, harvester_api_endpoints):
"""
Test creating images with no data.

Covers:
* Images-3: Create Image no data
"""
```

File: harvester_e2e_tests/apis/test_images.py [hunk 2]
```diff
@@ -27,12 +27,17 @@
 ]
 
 
+@pytest.mark.images_p1
+@pytest.mark.p1
 def test_list_images(admin_session, harvester_api_endpoints):
```
Comment (guangyee) [inline harvester_e2e_tests/apis/test_images.py:32]: Add test case cover docstring here.

**Review by guangyee — CHANGES_REQUESTED** [review]
The markers looks good. As for the test cases description, lets use Python docstring format.

**Comment by yarunachalam** [comment]
created PR 94 with all above comments

**Pull request closed without merging**